### ISSUE TITLE
[Feature] `push` and or `commit` input

### ISSUE THREAD

**Issue by nbrugger-tgm** [commitizen-tools/commitizen-action#15]
I think there are plenty of reasons why you do not want this action to commit or push the file, like adding the changelog to a github-release or posting the changelog into a comment in a PR or issue.

### Proposed inputs
```
push: if false the commited changes wont be pushed (default: true)
commit: if false changes wont be commited (default: true)
```

**Comment by Woile** [comment]
Sounds good! If you want to send a PR I'd be thankful. Otherwise next week I'll include it

**Comment by nbrugger-tgm** [comment]
Implemented in #16
